python/cudf_polars/cudf_polars/dsl/expr.py

```diff
@@ -1680,6 +1701,15 @@ def __init__(
                 f"with output type {self.dtype.id().name}"
             )
 
+    _KLEENE_MAPPING: ClassVar[
```

Comment:
Can we rename this to `_BOOL_KLEENE_MAPPING` since the bitwise mappings don't apply for other types?

Reply:
Sure.

Reply:
Done.